src/types/Revurdering.ts

```diff
@@ -1,33 +1,53 @@
 import { Behandling } from './Behandling';
 import { Beregning } from './Beregning';
+import { Periode } from './Fradrag';
 
 export interface Revurdering {
     id: string;
     status: RevurderingsStatus;
```

Comment:
```suggestion
export interface Revurdering<T extends RevurderingsStatus> {
    id: string;
    status: T;
```

Da kan man gjøre f.eks.
`type OpprettetRevurdering = Revurdering<RevurderingsStatus.OPPRETTET>`
og
```ts
export interface BeregnetInnvilget extends Revurdering<RevurderingsStatus.BEREGNET_INNVILGET> {
  beregninger: Beregninger;
}
```

Reply:
Ja det er sant. Men blir litt vanskeligare att bruke den som input parameter da, f. eks hvis en funktion tar in vilken som helst revurdering, så må man ju specificera `T`. Som man nödvändigtvis kanske inte vet. 😊 ?

Reply:
Skjønner. Tror man da kan gjøre `export interface Revurdering<T extends RevurderingsStatus = RevurderingsStatus> {`!

Reply:
Ser ut som det fungerade!